src/components/ListItem.jsx

```diff
@@ -69,29 +72,55 @@ export function ListItem({
 	}, [isChecked, itemId, listPath]);
 
 	const urgency = calculateUrgency(daysUntilNextPurchase, dateLastPurchased);
+	const label = { inputProps: { 'aria-label': `${name}` } };
 
 	return (
 		<>
 			<span>{message}</span>
-			<li className="ListItem">
-				<label htmlFor={`${name}`}>
-					<input
-						disabled={expired}
-						id={`${name}`}
-						type="checkbox"
-						checked={isChecked}
-						onChange={handleChange}
-					/>
-					{`${name}-${urgency}`}
-				</label>
-				<Button
-					onClick={handleClick}
-					variant="outlined"
-					startIcon={<DeleteIcon />}
+			<Card>
+				<li
+					className="ListItem"
+					style={{
+						display: 'flex',
+						alignItems: 'center',
+						justifyContent: 'space-between',
+					}}
 				>
-					Delete
-				</Button>
-			</li>
+					<div style={{ display: 'flex', alignItems: 'center' }}>
+						<Checkbox
+							{...label}
+							onChange={handleChange}
+							disabled={expired}
+							id={`${name}`}
+							checked={isChecked}
+						/>
+						<span style={{ marginRight: 'auto' }}>{name}</span>
+					</div>
+					<div style={{ display: 'flex', alignItems: 'center' }}>
+						<Chip
+							label={urgency}
+							sx={{
+								marginRight: 2,
+								backgroundColor:
+									urgency === 'soon'
+										? '#feff70'
```

Comment:
same as my comment on the other PR this would be better using ifs or a switch instead of nested ternarys